### PR TITLE
Add localization

### PR DESCRIPTION
**Pull request by ocallesp**
https://github.com/dotnet/interactive/issues/2093

### CODE REVIEW

File: src/polyglot-notebooks-vscode-insiders/package.json [hunk 1]
```diff
@@ -204,22 +205,22 @@
...
           "type": "string",
           "default": "https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json",
-          "description": "The NuGet source to use when acquiring the .NET Interactive tool."
+          "description": "%description.interactiveToolSource%"
         },
         "dotnet-interactive.minimumDotNetSdkVersion": {
           "type": "string",
           "default": "7.0",
-          "description": "The minimum required version of the .NET SDK."
+          "description": "%description.minimumDotNetSdkVersion%"
```
Comment (colombod) [inline src/polyglot-notebooks-vscode-insiders/package.json:218]: do we need to localise versions?

File: src/polyglot-notebooks-vscode-common/src/variableExplorer.ts [hunk 2]
```diff
@@ -134,6 +134,8 @@ class WatchWindowTableViewProvider implements vscode.WebviewViewProvider {
             }
         });
 
+        // Set the column names for the variable explorer table before rows are displayed
+        this.setColumnNames(['Name', 'Value', 'Type', 'Kernel', 'Actions']);
```
Comment (ocallesp) [inline src/polyglot-notebooks-vscode-common/src/variableExplorer.ts:138]: Testing. This will be localized in the next commit

**Comment by colombod** [comment]
it also needs to localise the screen reader message and hover text for the share button, the component logic is already done in #2812 , happy to discuss but that react code works, we just need to bring this in and wire with your work in the variableExplorer.ts.

**Review by colombod — CHANGES_REQUESTED** [review]
The react code is not correct

**Comment by ocallesp** [comment]
To fix: A task is missing. The pipeline references a task called 'OneLocBuild'. This usually indicates the task isn't installed, and you may be able to install it from the Marketplace: https://marketplace.visualstudio.com. (Task version 2, job 'OneLocBuild', step ''.)

File: global.json [hunk 3]
```diff
@@ -9,6 +9,6 @@
     "rollForward": "latestMinor"
   },
   "msbuild-sdks": {
-    "Microsoft.DotNet.Arcade.Sdk": "7.0.0-beta.23122.4"
```
Comment (colombod) [inline global.json:12]: we should not move to this version, @brettfo we need to stick to 7 until ready for .NET 8 release, right?

Comment (brettfo) [reply global.json:12]: Correct, keep this at 7.

**Comment by ocallesp** [comment]
Loc Team request https://ceapex.visualstudio.com/CEINTL/_workitems/edit/811923

**Comment by ocallesp** [comment]
We just got assigned the package Id: LCL-JUNO-PROD-INTERACTIVE

**Pull request closed without merging**